src/stores/map.ts

```diff
@@ -117,6 +120,64 @@ export const useMapStore = defineStore("map", () => {
     }
   }
 
+  /**
+   * Create manual clusters for a given set of geographic spaces.
+   * @param {FeatureCollection} geoJsonSpaces - A GeoJSON FeatureCollection of spaces.
+   * @returns {FeatureCollection} - A new FeatureCollection containing clustered and original features.
+   */
+  function createManualClusters(geoJsonSpaces: FeatureCollection): FeatureCollection {
```

Comment:
Looks like this exact function does not need the parameter because it always references the geoJsonSpaces from the store? There're of course multiple ways to organise this code, no extra-strong opinion here :)

Reply:
Yeah i'm also not happy with this one because also need it in the MapboxMap.vue to generate the first building clusters and also need it in the store to update the filtered occupancy and slugs